dev/tl/helmfile.yaml

```diff
@@ -28,23 +28,23 @@ releases:
     namespace: tl   # target namespace
     createNamespace: true
     chart: datarepo-helm/datarepo-api   # the chart name
-    version: 0.0.25    # chart version
+    version: 0.0.57    # chart version
     missingFileHandler: Warn
     values:
       - datarepo-api.yaml   # Value files passed via --values
   - name: tl-jade-datarepo-ui   # name of this release
     namespace: tl   # target namespace
     createNamespace: true
     chart: datarepo-helm/datarepo-ui   # the chart name
-    version: 0.0.14    # chart version
+    version: 0.0.50    # chart version
     missingFileHandler: Warn
     values:
```

Comment:
Latest is now 0.0.61 for API and 0.0.53  for UI. https://github.com/broadinstitute/datarepo-helm/blob/master/charts/datarepo/Chart.lock

Totally feel free to ignore, but I recommend removing the "version" line for API and UI. If you remove it, it will default to the latest. Since these both change so often it's hard to keep up!

Reply:
Oh that's good to know!